### DISCUSSION TITLE
H&M - 4 new Side Missions

### DISCUSSION THREAD

**Discussion by BadDad1776** [Vdauphin/HeartsAndMinds#1579] [Show and tell]
decided i am gonna remake these missions so they dont require dependancy will upload when completed 

Destroy IED factory - Search for and destroy the IED factory. (Use one M112 explosive to destroy the IED Cache).
Destroy the mortar camp -Search for and destroy the Mortar Camp. (Destroy the Mortar Cache and mortar tubes with explosives).
Destroy the AA camp - Search for and destroy the Anti-Air Camp. (Destroy the Anti-Air guns at the site with explosives).
Plane Crash Site -  A plane carrying a secure data terminal has gone down... your objective is to secure the crash site and retrieve the data.
If anyone makes changes or has suggestions for changes please feel free to let me know.

**Comment by Jaz0314** [comment]
Did you ever get around to making them? I'm very interested in some new side missions for my group. Also as an idea for one, how about a base defense that starts with a 10-15min timer until the enemy starts attacking the players main base.
